### DISCUSSION TITLE
Trying to add values in Model Atrributes on Create and Update

### DISCUSSION THREAD

**Discussion by Nav-Chouhan** [Laravel-Backpack/community-forum#444] [Q&A (Help)]
I have a table of customers where we have 3 columns Country, State, and City. Now I am adding or updating the value in these 3 fields but they are not getting populated.

Note: These are select_from_ajax fields on form.

May I know the reason why this is happening?

Thanks in advance.

**Comment by karandatwani92** [comment]
Hello @Nav-Chouhan,

Please check field name and table column name are same and fillable in the model. Also, go through this [link](https://backpackforlaravel.com/docs/5.x/crud-fields#select2_from_ajax-pro) to reconfirm required steps.

If the problem persist, share table column names and the code here to identify the issue.

**Comment by Nav-Chouhan** [comment]
I am closing this ticket.

I have a better explanation for my issue https://github.com/Laravel-Backpack/community-forum/discussions/445

@karandatwani92 Thanks for guiding.
